pkg/scaffold/project/project.go

```diff
@@ -87,7 +87,7 @@ func (Project) repoFromGopathAndWd(gopath string, getwd func() (string, error))
 			// '$GOPATH/src/kubernetes-sigs/controller-tools' becomes 'kubernetes-sigs/controller-tools'
 			repo := ""
 			for wd != goSrc {
-				repo = filepath.Join(filepath.Base(wd), repo)
+				repo = filepath.ToSlash(filepath.Join(filepath.Base(wd), repo))
```

Comment:
shouldn't the filepath.ToSlash be on `repo`?

Reply:
The final output should be normalized to forward slash to match with Go import statements. On Windows and filepath operation will flip it back.

Reply:
then this should just be `path.Join`, and dispense with the `filepath` stuff entirely (or we should be calling `filepath.Rel` and then `ToSlash` instead of this for loop, but that's a different story).

Reply:
Fixed. In the other PR, the initial 3 filepath changes were also the only necessary ones (they compared with OS filepaths). The rest were okay functionally but I assumed the preference was for them to be the same. LMK what you want?